app-frontend/config/webpack/environments/production.js

```diff
@@ -17,6 +17,20 @@ module.exports = function (_path) {
             filename: '[name].[chunkhash].js'
         },
         plugins: [
+        // Return error code when build fails.

```

Comment:
I think there's an argument to be made to keep this change as opposed to adding another dependency though too, even if it is small